test/test_tensorexpr_pybind.py

```diff
@@ -206,5 +206,104 @@ def forward(self, x, y):
         correct = TestModule().forward(x, y)
         np.testing.assert_allclose(res.numpy(), correct.numpy(), atol=1e-5)
 
+    @unittest.skipIf(not LLVM_ENABLED, "LLVM backend not enabled")
+    def test_kernel_with_t(self):
+        def f(a):
+            return a.t()
+
+        device, size = 'cpu', (3, 4)
+        x = torch.rand(size, device=device)
+
+        graph_str = """
+graph(%a.1 : Float(3, 4, strides=[4, 1], requires_grad=0, device=cpu)):
+  %3 : Float(4, 3, strides=[4, 1], requires_grad=0, device=cpu) = aten::t(%a.1)
+  return (%3)
+        """
+        graph = torch._C.parse_ir(graph_str)
+
+        kernel = torch._C._te.TensorExprKernel(graph)
+        res1 = kernel.run((x,))
+        res2 = kernel.fallback((x,))
+        correct = f(x)
+        np.testing.assert_allclose(res1.numpy(), correct.numpy(), atol=2e-3)
```

Comment:
Nit: I know we use this `2e-3` accuracy in some existing tests, but I think we really should be more accurate than this :) Can we change it to `1e-5`?